app-text/mtafsir/mtafsir-0.2.recipe

```diff
@@ -0,0 +1,91 @@
+SUMMARY="Application that provide Interpretation of the holy Quran"
+DESCRIPTION="Provide interpretation of the holy quran
+* Tafsir fi karimir rohmaan
+* Tafsir As Sa'di
+* Tafsir at tabari
+* Tafsir al kortoby
+* Minister of religion indonesia
+* etc
+."
+HOMEPAGE="https://github.com/zakariakov/mtafsir"
+COPYRIGHT="2015-2018 Abou Zakariya"
+LICENSE="GNU GPL v3"
+REVISION="1"
+SOURCE_URI="https://github.com/zakariakov/mtafsir/archive/$portVersion.tar.gz"
+CHECKSUM_SHA256="1e0dfbee7a6ef8cda68a070aeaba2075e5d9be6e1c6428d55925f1bb19d47a6a"
+SOURCE_FILENAME="mtafsir-$portVersion.tar.gz"
+ADDITIONAL_FILES="mtafsir.rdef.in"
+
+ARCHITECTURES="!x86_gcc2 x86 x86_64"
+if [ "$targetArchitecture" = x86_gcc2 ]; then
+SECONDARY_ARCHITECTURES="x86"
+fi
+
+PROVIDES="
+	mtafsir$secondaryArchSuffix = $portVersion
+	app:Mtafsir = $portVersion
+	"
+REQUIRES="
+	haiku$secondaryArchSuffix
+	ayat_tafasir_data
+	lib:libGL$secondaryArchSuffix
```

Comment:
Is it really needed?

Reply:
@extrowerk  it is depend on ayat_tafasir_data recipe.. yes.. it contain data for mtafsir to avoid duplication